crates/noirc_evaluator/src/ssa_refactor/ir/function.rs

```diff
@@ -1,11 +1,20 @@
 use std::collections::HashSet;
 
+use crate::brillig::brillig_gen::BrilligGen;
+
 use super::basic_block::BasicBlockId;
 use super::dfg::DataFlowGraph;
 use super::map::Id;
 use super::types::Type;
 use super::value::ValueId;
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+pub(crate) enum RuntimeType {
+    // A noir function
+    Normal,
+    // An unsafe function
+    Unsafe,
```

Comment:
```suggestion
    /// A normal function in Noir to be compiled into SSA and ACIR
    Normal,
    /// An unconstrained function to be compiled to Brillig
    Unconstrained,
```

Reply:
On second thought, since this is called `RuntimeType` perhaps we can cut the middle-man entirely and specify the name of the primary runtime:
```suggestion
    /// Most functions are compiled to SSA then ACIR
    Acir,
    /// Unconstrained functions are compiled to Brillig
    Brillig,
```